guacamole/src/main/frontend/src/app/index/indexModule.js

```diff
@@ -17,22 +17,37 @@
  * under the License.
  */
 
+require('angular-module-shim.js');
```

Comment:
What are these imports doing up at the top here? I'm guessing these are just global runtime requirements?

Reply:
Yep, exactly.